src/helpers.php

```diff
@@ -29,9 +29,10 @@ function abort($code, $message = '', array $headers = [])
     /**
      * Get the available container instance.
      *
-     * @param  string|null  $make
+     * @template T
+     * @param  class-string<T>  $make
```

Comment:
this doc isn;t accurate i'm afraid. you can also put non-class-strings in here, like `hash` or `db.connection` (disclaimer: probs something like that :trollface:)

Reply:
I think instead we should leave this to larastan and similar tools to implement cc @nunomaduro